### PR TITLE
Fix/mobile layout

### PR DESCRIPTION
**Pull request by BBCmisha**

### CODE REVIEW

File: src/views/ValidatorItem.vue [hunk 1]
```diff
@@ -3,8 +3,10 @@
     <div class="view-main__title-wrapper">
       <BackButton :text="'Validators'" />
       <h2 class="view-main__title">Validator</h2>
-      <div class="validator__address">
-        <p class="view-main__subtitle">{{ validator?.operatorAddress }}</p>
+      <div class="view-main__subtitle-wrapper">
+        <p class="view-main__subtitle fs-cut">
+          {{ validator?.operatorAddress }}
```
Comment (lukachi) [inline src/views/ValidatorItem.vue:8]: what if operationAddress will not be there? you will display undefined?
handle this place -> you can create computed property and check there

File: src/views/Transactions.vue [hunk 2]
```diff
@@ -93,8 +71,7 @@ export default defineComponent({
 
         await filterTransactions(page.value)
       } catch (e) {
-        console.error(e.message)
-        handleError(e)
+        handleError(e as Error)
```
Comment (lukachi) [inline src/views/Transactions.vue:74]: catch (e: Error) {
    handleError(e)
}

File: src/styles/_mixins.scss [hunk 3]
```diff
@@ -48,13 +48,13 @@
...
-  xm: 1200px,
-  lg: 1440px,
-  xl: 1920px,
+  medium: 1024px,
+  x-medium: 1200px,
+  large: 1440px,
+  x-large: 1920px,
 ) !default;
 
 @mixin respond-to($media) {
```
Comment (lukachi) [inline src/styles/_mixins.scss:60]: need to talk

File: src/components/pagination/pagination.vue [hunk 4]
```diff
@@ -1,96 +1,93 @@
...
-<style scoped lang="scss">
-.pagination {
-  margin: 4rem 0 6rem 0;
-  &__buttons {
+<style lang="scss">
+.pagination-wrapper {
+  display: flex;
+  justify-content: center;
+
+  li {
```
Comment (lukachi) [inline src/components/pagination/pagination.vue:47]: class

File: src/components/pagination/pagination.vue [hunk 5]
```diff
@@ -1,96 +1,93 @@
...
+    background: #fff;
+    border: 0.1rem solid var(--clr__action);
+    border-radius: 0.4rem;
+    margin: 0 0.4rem;
+    min-width: 2.6rem;
+    height: 3.6rem;
+  }
+
+  button {
```
Comment (lukachi) [inline src/components/pagination/pagination.vue:56]: class

File: src/views/Transactions.vue (hunk 2 above)
Comment (BBCmisha) [reply src/views/Transactions.vue:74]: Catch clause variable type annotation must be 'any' or 'unknown' if specified.Vetur(1196)

File: src/styles/_mixins.scss (hunk 3 above)
Comment (BBCmisha) [reply src/styles/_mixins.scss:60]: add breakpoint for tablet (768px)

File: src/components/pagination/pagination.vue [hunk 6]
```diff
@@ -1,96 +1,93 @@
...
+  .DotsHolder {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+
+    svg,
+    path {
+      color: var(--clr__action);
     }
```
[the remark this answers is not in the document]
Comment (BBCmisha) [reply src/components/pagination/pagination.vue:91]: @ hennge / vue3-pagination package is used, this is the only way I can interact with elements of this pagination

File: src/components/pagination/pagination.vue [hunk 7]
```diff
@@ -1,96 +1,93 @@
...
-    justify-content: center;
-    gap: 0.8rem;
+    border: none;
+    margin: 0;
+    padding: 1rem;
   }
-  &__button {
-    padding: 0.6rem 1.2rem;
+
+  .Page {
```
[the remark this answers is not in the document]
Comment (BBCmisha) [reply src/components/pagination/pagination.vue:64]: @ hennge / vue3-pagination package is used, this is the only way I can interact with elements of this pagination

File: src/components/pagination/pagination.vue (hunk 5 above)
Comment (BBCmisha) [reply src/components/pagination/pagination.vue:56]: @ hennge / vue3-pagination package is used, this is the only way I can interact with elements of this pagination

File: src/components/pagination/pagination.vue (hunk 4 above)
Comment (BBCmisha) [reply src/components/pagination/pagination.vue:47]: @ hennge / vue3-pagination package is used, this is the only way I can interact with elements of this pagination

File: src/views/ValidatorItem.vue (hunk 1 above)
Comment (BBCmisha) [reply src/views/ValidatorItem.vue:8]: done

File: src/views/Home.vue [hunk 8]
```diff
@@ -1,9 +1,7 @@
 <template>
-  <div class="home">
-    <div class="home__wrapper">
-      <InfoPanel />
-      <Latest />
-    </div>
+  <div class="view-main">
```
[the remark this answers is not in the document]
Comment (BBCmisha) [reply src/views/Home.vue:2]: changed class to container

File: src/views/ValidatorItem.vue [hunk 9]
```diff
@@ -3,11 +3,11 @@
     <div class="view-main__title-wrapper">
       <BackButton :text="'Validators'" />
       <h2 class="view-main__title">Validator</h2>
-      <div class="view-main__subtitle-wrapper">
+      <div class="view-main__subtitle-wrapper" v-if="validator">
```
Comment (lukachi) [inline src/views/ValidatorItem.vue:6]: if validator is Object ({}) it will be always true

**Pull request merged**